### ISSUE TITLE
The audio suddenly stop then the Apple logo started to appear

### ISSUE THREAD

**Issue by h3d-haivq** [Goldfish64/AudioPkg#15]
I'm using MSI Tomahawk B450, with Realtek ALC892 audio chip

**Comment by Goldfish64** [comment]
I don't quite understand what you are trying to say. Is the audio cutting out before playback is finished? AMD boards may not fully follow the HDA spec, and I don't have such hardware, so there are bound to be issues.

**Comment by h3d-haivq** [comment]
> I don't quite understand what you are trying to say. Is the audio cutting out before playback is finished? AMD boards may not fully follow the HDA spec, and I don't have such hardware, so there are bound to be issues.

Yeah the audio is played before the Apple boot be loaded up and it's cut out before the playback is finished. Sorry for my terrible description. Is there anyway to fix that problem? Btw how can I run the BootChimeCfg, I tried to put it both inside and outside the clover driver directory, nothing appear like a boot selection in the clover booting screen.
